nixos/modules/services/networking/eternal-terminal.nix

```diff
@@ -0,0 +1,96 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.eternal-terminal;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.eternal-terminal = {
+
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Enable the Eternal Terminal server.
+        '';
+      };
+
+      port = mkOption {
+        default = null;
+        type = types.nullOr types.int;
+        description = ''
+          The port the server should listen on. Will use the server's default (2022) if not specified.
+        '';
+      };
+
+      verbosity = mkOption {
+        default = 0;
+        type = types.int;
+        description = ''
+          The verbosity level (0-9).
+        '';
+      };
+
+      silence = mkOption {
+        default = 0;
```

Comment:
as this is a true/false toggle this should be of type boolean and then transformed with `if cfg.silent then "1" else "0"`.

Reply:
Yep.